package.json

```diff
@@ -19,6 +19,12 @@
     "url": "https://github.com/sgmap/codes-postaux/issues"
   },
   "homepage": "https://github.com/sgmap/codes-postaux",
+  "files": [
+    "LICENSE.MIT.txt",
+    "codes-postaux.json.gz",
+    "postinstall.js",
+    "index.js"
+  ],
```

Comment:
Blacklist probably decreases risk of bad publishes. I'll revert this set of adds.